src/main.c

```diff
@@ -134,17 +159,17 @@ void hide_window (GromitData *data)
       /* save grab state of each device */
       GHashTableIter it;
       gpointer value;
-      GromitDeviceData* devdata; 
+      GromitDeviceData* devdata;
```

Comment:
In general, I preserve WS, but these are _trailing_ WS that IMO should not be there.

Preserving them would be complicated because I have my editor (emacs) set up to automatically remove these. 

Not sure if it makes sense to put them all back in...